addons/web/static/src/env.js

```diff
@@ -74,6 +75,10 @@ export async function startServices(env) {
             const namedService = Object.assign(Object.create(service), { name });
             toStart.add(namedService);
         } else {
+            if (isStartingService) {
```

Comment:
commit to be validated by framework

Reply:
it's in its own commit. It's necessary otherwise there are several services mounted at once, and some tours fail because closing a livechat still show the livechat button from other instances below itself.